lib/utils/base_env.dart

```diff
@@ -19,6 +21,12 @@ class BaseEnv {
           grpcInfo: GRPCInfo(
             host: grpcUrl ?? envGrpcUrl,
             port: int.parse(grpcPort ?? envGrpcPort),
+            credentials: ChannelCredentials.secure(
+              onBadCertificate: (cert, host) {
+                debugLog("host: $host, cert: $cert");
+                return true;
+              },
```

Comment:
this is just to make sure the `https` communication ignores any certificate errors, just for testing purposes, this should not be part of the production code though